File: C++/disk.cpp
#include <iostream>
#include <string>
using namespace std;
class disk{
  
    private:
        string tipe; //deklarasi Attribute
        string kapasitas; //deklarasi Attribute
        int harga; //deklarasi Attribute
    public:
        // Constructor kosong
        disk(){

        }

        disk(string type, string capacity, int price){
            this->tipe = type;
            this->kapasitas = capacity;
            this->harga = price;
        }
        // Get dan Set
        void setTipe(string type){
            this->tipe = type;
        }
        string getTipe(){
            return this->tipe;
        }
        void setKapasitas(int capacity){
            this->kapasitas = capacity;
        }
        string getKapasitas(){
            return this->kapasitas;
        }
        void setHarga(int price){
            this->harga = price;
        }
        int getHarga(){
            return this->harga;
        }
      
        //keluaran
        void printDisk(){
            cout << "---- DISK ---" << endl;
            cout << "Type: " << this->tipe << endl;
            cout << "Capacity: " << this->kapasitas << endl;
            cout << "Price: " << this->harga << endl;
        }
        
        ~disk(){}
};

File: Java/Disk.java
public class Disk{
    
    private String kapasitas; //deklarasi Attribute
    private int harga; //deklarasi Attribute
    private String tipe; //deklarasi Attribute

    // Constructor
    Disk(){

    }
    public Disk(String kapasitas, int harga, String tipe) {
        this.kapasitas = kapasitas;
        this.harga = harga;
        this.tipe = tipe;
    }
    
    // Get dan Set
    public String getKapasitas() {
        return kapasitas;
    }
    public void setKapasitas(String kapasitas) {
        this.kapasitas = kapasitas;
    }
    public int getHarga() {
        return harga;
    }
    public void setHarga(int harga) {
        this.harga = harga;
    }
    public String getTipe() {
        return tipe;
    }
    public void setTipe(String tipe) {
        this.tipe = tipe;
    }
    
    //keluaran
    public void printDisk(){
        System.out.println("---- DISK ----");
        System.out.println("Type: " + this.tipe);
        System.out.println("Capacity: " + this.kapasitas);
        System.out.println("Price: " + this.harga);
    }
}